MetaMask/metamask-extension
# Fix opening of speedup popover in activity list (in the popup view)

Fixes https://github.com/MetaMask/metamask-extension/issues/20186

## Explanation

The bug is that, when metamask is opened in the popup view, clicking "Speed up" on an activity list items opens the transaction details and not the speed up popover.

The problem was that the elements containing the transaction cost data were too heigh, and so were overlapping the speed up button and receiving the clicks instead of the the button.

The fix is to minimize the height of those elements to fit their content.

## Screenshots/Screencaps

### Before

![Screenshot from 2023-07-26 10-03-20](https://github.com/MetaMask/metamask-extension/assets/7499938/a4bf5e72-915c-4216-9b90-d6feed5c1fdd)


### After

![Screenshot from 2023-07-26 10-02-59](https://github.com/MetaMask/metamask-extension/assets/7499938/7921d534-01d2-48b0-96e0-b6d266656964)


https://github.com/MetaMask/metamask-extension/assets/7499938/5f9071e2-ac16-45ce-be38-96d8f9cea585

## Manual Testing Steps

1. Open MetaMask in the popup view
2. Create a transaction on goerli
3. wait for the speed up button, and then click it

The speed up popup should open

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
